src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -1026,7 +1039,8 @@ namespace AppInstaller::Repository
                 AICLI_LOG(Repo, Info, << "Named source no longer found. Source was tombstoned: " << details.Name);
                 return {};
             }
-            if (ShouldUpdateBeforeOpen(*source))
+            if (source->Restricted != true && 

```

Comment:
nit: binary operators on booleans are more difficult to understand (for me at least)
```
source->Restricted != true
```
vs
```
!source->Restricted
```